Classes/common/touchdb/TDBlobStore.m

```diff
@@ -198,17 +202,53 @@ - (NSInteger)deleteBlobsExceptWithKeys:(NSSet*)keysToKeep withDatabase:(FMDataba
         if ([[NSFileManager defaultManager] removeItemAtPath:blobPath error:&thisError]) {
```

Comment:
If the file doesn't exist, does this method return `YES`? I ask as we can also remove the database row for the key if the file didn't exist in the first place (for some reason).